client/layout/masterbar/logged-in.jsx

```diff
@@ -96,6 +96,7 @@ export default React.createClass( {
 					{ this.translate( 'New Post' ) }
 				</Publish>
 				<Item
+					tipTarget="me"
```

Comment:
@folletto hmmm — we deliberately chose something generic so it could be used by other components as well.